package.json

```
@@ -1,6 +1,13 @@
 {
   "name": "developerdao-job-board",
 	"private": true,
+	"scripts": {
+		"dev": "yarn workspace @developer-dao-job-board/client dev",
+    "build": "yarn workspace @developer-dao-job-board/client next build",
+    "start": "yarn workspace @developer-dao-job-board/client next start",
+    "lint": "yarn workspace @developer-dao-job-board/client next lint",
+    "type-check": "yarn workspace @developer-dao-job-board/client tsc --noEmit"
```

Indentation seems to be inconsistent, could be my package.json I made originally that is weird 😅 we might want to add a `.editorconfig` or `pre-commit hook` with prettier to avoid that in the future

---

I have a `prettier` branch that I didn't quite finish that I need to get back to, but definitely agree there. Formatting is wildly inconsistent in this codebase but we'll get that sorted soon!

---

Echoing some feedback from someone in Discord, the root `package.json` scripts should mostly be referencing scripts defined in each package:

```suggestion
    "build": "yarn workspace client build",
    "start": "yarn workspace client start",
    "lint": "yarn workspace client lint",
    "type-check": "yarn workspace client type-check"
```

This gives us a nice cascade where individual packages define scripts for their own concerns, and then we can promote those to be executed in the root when necessary/useful.

---

Oh mb! My intention was to run the scripts that the packages have. I was copy-pasta'ing those and eventually forgot or idk I don't remember what happened 😅 Thanks for that.

---

> I have a prettier branch that I didn't quite finish that I need to get back to, but definitely agree there. Formatting is wildly inconsistent in this codebase but we'll get that sorted soon!

Agreed, we really need to have the consistency

---

> Indentation seems to be inconsistent, could be my package.json I made originally that is weird 😅 we might want to add a .editorconfig or pre-commit hook with prettier to avoid that in the future

That would be cool. I'm looking forward to that. I'm learning things about CI here XD